pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -257,6 +258,18 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 		return reconcile.Result{}, nil
 	}
 
+	// Only allows live migration on SD managed clusters
+	inManagedCluster, err := util.IsRunningInManagedCluster(ctx, r.client)
+	if err != nil {
+		log.Println(err)
+		return reconcile.Result{}, err
+	}
+	if !inManagedCluster && operConfig.Spec.Migration != nil && operConfig.Spec.Migration.Mode == operv1.LiveNetworkMigrationMode {
+		r.status.SetDegraded(statusmanager.OperatorConfig, "MergeClusterConfig",
+			"invalid configuration: network type live migration is not supported on self-managed clusters. Use 'oc edit network.operator.openshift.io cluster' to fix.")
```

Comment:
I would be more vague in this error given that in this case the user must have modified the operator config itself and not the cluster config so the resolution would not be `oc edit network.operator.openshift.io cluster' to fix.`

Reply:
is the operator.config itself modifiable by the user? I always forget which CRD is for users to modify and which isn't

Reply:
user should only modify `network.config.openshift.io` and not `network.operator.openshift.io`.

Jaime, youre correct, I will make a new PR.